x-pack/platform/plugins/shared/onechat/public/embeddable/embeddable_conversation.tsx

```diff
@@ -23,7 +23,10 @@ export const EmbeddableConversationInternal: React.FC<EmbeddableConversationInte
     background-color: ${euiTheme.colors.backgroundBasePlain};
   `;
 
+  const headerHeight = `calc(${euiTheme.size.xl} * 2)`;
   const headerStyles = css`
+    display: flex;
+    height: ${headerHeight};
```

Comment:
Stops the header from changing in height when we go from /new conversation to a conversation which has a title. A conversation with a title will render a button in the middle of the header which pushes the height higher. Here, we're setting the fixed height to be the same as the fixed height that we use in `conversations_view.`